string manufacturer
string typeNumber
string serialNumber
---
